app/workers/push_notifications/deliver_worker.rb

```diff
@@ -0,0 +1,18 @@
+module PushNotifications
+  class DeliverWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :medium_priority,
+                    retry: 10,
+                    lock: :until_expired,
+                    lock_ttl: 30,
+                    on_conflict: :reject
+
+    def perform
+      # Deliver all pending Push Notifications
+      Rpush.push
+      # Callback for feedback (see `config/initializers/rpush.rb`)
+      Rpush.apns_feedback
```

Comment:
Related to the suggestion in the other worker above: Since this would be called periodically in the scheduled job we can get rid of it here.